flake.nix

```diff
@@ -458,61 +478,85 @@
                 "cargo test --workspace --release --locked --verbose";
             });
 
-            kusama-picasso-karura = let
-              config = (pkgs.callPackage
-                ./scripts/polkadot-launch/kusama-local-picasso-dev-karura-dev.nix {
-                  polkadot-bin = polkadot-node;
-                  composable-bin = composable-node;
-                  acala-bin = acala-node;
-                }).result;
-              config-file = writeTextFile {
-                name = "kusama-local-picasso-dev-karura-dev.json";
-                text = "${builtins.toJSON config}";
+            kusama-picasso-karura =
+              let
+                config = (pkgs.callPackage
+                  ./scripts/polkadot-launch/kusama-local-picasso-dev-karura-dev.nix
+                  {
+                    polkadot-bin = polkadot-node;
+                    composable-bin = composable-node;
+                    acala-bin = acala-node;
+                  }).result;
+                config-file = writeTextFile {
+                  name = "kusama-local-picasso-dev-karura-dev.json";
+                  text = "${builtins.toJSON config}";
+                };
+              in
+              writeShellApplication {
+                name = "kusama-picasso-karura";
+                text = ''
+                  cat ${config-file}
+                  ${packages.polkadot-launch}/bin/polkadot-launch ${config-file} --verbose
+                '';
               };
-            in writeShellApplication {
-              name = "kusama-picasso-karura";
-              text = ''
-                cat ${config-file}
-                ${packages.polkadot-launch}/bin/polkadot-launch ${config-file} --verbose
-              '';
-            };
 
             default = packages.composable-node;
           };
 
           devShells = rec {
-            developers = mkShell (common-attrs // {
-              buildInputs = with packages; [
-                bacon
-                google-cloud-sdk
-                grub2
-                jq
-                lldb
-                llvmPackages_latest.bintools
-                llvmPackages_latest.lld
-                llvmPackages_latest.llvm
-                mdbook
-                nix-tree
-                nixpkgs-fmt
-                openssl
-                openssl.dev
-                pkg-config
-                qemu
-                rnix-lsp
-                rust-nightly
-                taplo
-                wasm-optimizer
-                xorriso
-                zlib.out
-              ];
-              NIX_PATH = "nixpkgs=${pkgs.path}";
+
+            junod = pkgs.callPackage ./.nix/junod.nix { };
```

Comment:
I think you accidentally added this as a devshell instead of as a package

Reply:
bad merge